src/DependencyInjection/DI.Specification.Tests/src/Microsoft.Extensions.DependencyInjection.Specification.Tests.csproj

```diff
@@ -12,6 +12,7 @@
     -->
     <EnableApiCheck>false</EnableApiCheck>
     <IsTestProject>false</IsTestProject>
+    <IsUnitTestProject>false</IsUnitTestProject>
```

Comment:
Is IsUnitTestProject an Arcade thing?  I wonder if we can replace usages of IsTestProject. They seem redundant.

Reply:
Yes - this is an arcade thing. They map tests into 3 categories (unit test, integration test, perf test). I'm not sure what's different about them.